package.json

```diff
@@ -29,14 +29,18 @@
     "lib": "./lib"
   },
   "dependencies": {
+    "@digitalbazaar/ezcap-express": "^3.2.0",
+    "@digitalbazaar/http-digest-header": "^1.0.0",
     "ajv": "^6.0.0",
     "base58-universal": "^1.0.0",
+    "express-async-handler": "^1.1.4",
```

Comment:
Missing `esm` and other related field updtes for index vs main.

Reply:
What are the "other related field updates"?

Reply:
The ones we haven't figured out yet.  Whatever combination of "main", "module", "exports", etc are needed to make cjs + esm work properly.  Looks like just "main" is there now and would load `lib/index.js`, but nothing points at `lib/main.js` for ESM use.

Reply:
Add `module` for now to address the main point. Not dealing with the larger/systemic issue at this time.